File: Klasa 3/py/instrukcje warunkowe.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  instrukcje warunkowe.py
#  



def main(args):
    a = int(input("Podaj pierwszą liczbę: "))
    b = int(input("Podaj drugą liczbę: "))
    c = int(input("Podaj trzecią liczbę: "))
    d = 0
    print(a, b, c)
    if a>b and a>c:
        d=a
    elif b>a and b>c:
        d=b
    elif c>a and c>b:
        d=c
    print ("Największa liczba to: ", d)
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: Klasa 3/cpp/instrukcje warunkowe.cpp
/*
 * instrukcje warunkowe.cpp
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int a, b, c;
    a = b = c = 0;
    cout << "Podaj pierzwsza liczbę: ";
    cin >> a;
    cout << "Podaj drugą liczbę: ";
    cin >> b;
    cout << "Podaj trzecią liczbę: ";
    cin >> c;
    if(a>b and a>c) cout << "Największa liczba to: " << a << endl;
    if(b>a and b>c) cout << "Największa liczba to: " << b << endl;
    else cout << "Największa liczba to: " << c << endl;
    
	return 0;
}

